gradle/libs.versions.toml

```diff
@@ -1,13 +1,13 @@
 [versions]
-agp = "8.3.2"
+agp = "8.4.0"
 kotlin = "1.9.0"
-coreKtx = "1.13.0"
+coreKtx = "1.13.1"
 junit = "4.13.2"
 junitVersion = "1.1.5"
 espressoCore = "3.5.1"
 lifecycleRuntimeKtx = "2.7.0"
 activityCompose = "1.9.0"
-composeBom = "2024.04.01"
+composeBom = "2024.05.00"
 navigationCompose = "2.7.7"
```

Comment:
Tak z ciekawości to podbijanie tych wersji było do czegoś niezbędne czy tak programowo dałeś żeby były nowsze?

Reply:
po prostu intelijej mi podpowiedział ze jest mozliwosc dodania nowej wersji, chyba w warningach przy comitach

Reply:
Ma sens i chyba faktycznie warto podbijać, a działało Ci z tym `agp = "8.4.0"`?

Reply:
![image](https://github.com/Nepommuck/SusGameFrontend/assets/100963527/283a6365-6a1a-4107-9af0-e994440f8262)
takie powiadomienie wyskakuje, jak zainstaluje to automatycznie zmienia na 8.4.0 i wszystko smiga

Reply:
Hmm ciekawe, u mnie nie ma. Może to przez wersję IDE, spróbuję zaktualizować, bo mam `2023.3.1`